### PR TITLE
[typo]update spark build doc

### PR DESCRIPTION
**Pull request by wangbo**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by luzhijing** [comment]
LGTM

File: docs/zh-CN/extending-doris/spark-doris-connector.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ sh build.sh 2.3.4 2.11 ## spark 2.3.4, scala 2.11
 sh build.sh 3.1.2 2.12 ## spark 3.1.2, scala 2.12
 
 ```
+> 注：如果你是从 tag 检出的源码，则可以直接执行 `sh build.sh --tag`，而无需指定 spark 和 scala 的版本。因为 tag 源码中的版本是固定的。
```
Comment (dataroaring) [inline docs/zh-CN/extending-doris/spark-doris-connector.md:54]: Could you update the en doc?

Comment (wangbo) [reply docs/zh-CN/extending-doris/spark-doris-connector.md:54]: Sure. This is a good suggestion.

**Review by dataroaring — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]

**Review by hf200012 — COMMENTED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**